v2/internal/schemaregistry/config.go

```diff
@@ -0,0 +1,7 @@
+package schemaregistry
+
+type Config struct {
+	Endpoint string `env:"KP_SCHEMA_REGISTRY_ENDPOINT" required:"true"`
+	Username string `env:"KP_SCHEMA_REGISTRY_USERNAME" default:""`
```

Comment:
wouldnt this also be required? can help reduce mistakes?

Reply:
in some cases you don't need username and passwords (like while working locally)